src/Pages/MainPage.jsx

```diff
@@ -0,0 +1,233 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { requestSignup } from '../Store/Actions/auth';
+import Icon from '../components/Icon.jsx';
```

Comment:
Paths are case-sensitive. 
```suggestion
import Icon from '../Components/Icon';
```

Reply:
This should also fix the warning in windows.